cloud/deployment/inspect/inspect.go

```diff
@@ -414,3 +455,19 @@ func getTemplate(formattedDeployment *FormattedDeployment) FormattedDeployment {
 
 	return template
 }
+
+func GetDefaultWorkerType(taskPodNodePoolID, clusterID string, platformCoreClient astroplatformcore.CoreClient) (string, error) {
+	var defaultWorkerType string
+	cluster, err := deployment.CoreGetCluster("", clusterID, platformCoreClient)
+	if err != nil {
+		return "", err
+	}
+	nodePools := *cluster.NodePools
```

Comment:
It's a slice and not a pointer of slices, so I'm kinda confused why we're dereferencing here

Reply:
nvm vscode gave me the wrong definition, these are all correct